normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
+        schema = json.loads(
+            open('normandy/recipes/static/actions/console-log/package.json').read()
```

Comment:
@rehandalal Right now I'm just pulling in the `console-log` schema to validate against. This one is a bit too simple for all the complex logic in `validate_arguments`. I can switch it to the `show-heartbeat` schema orrrrr is it bad practice to depend on these files and I should just mock up an entirely fake one?